python_setup.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# Set up Python environment.
+# Following the instructions at:
+# https://github.com/cfpb/development/blob/master/guides/installing-python.md
+
+# Specify the default version of Python that we use.
+export PYTHON3_VER="3.6.9"
+
+install_pyenv() {
+    echo "$(tput setaf 3)$(tput setab 4)...Installing pyenv$(tput sgr 0)"
+
+    brew install pyenv pyenv-virtualenv
+
+    echo "# Python and pyenv" >> ${HOME}/.zshenv
+    echo export PYENV_ROOT="${HOME}/.pyenv"
+    echo export PATH="$PYENV_ROOT/bin:$PATH"
+    echo eval "$(pyenv init --path)" >> ${HOME}/.zshenv
+    echo eval "$(pyenv virtualenv-init -)" >> ${HOME}/.zshenv
```

Comment:
A few fixes to these setups got a Big Sur user going:

```suggestion
    echo "# Python and pyenv" >> ${HOME}/.zshenv
    echo export PYENV_ROOT="${HOME}/.pyenv" >> ${HOME}/.zshenv
    echo export PATH="$PYENV_ROOT/bin:$PATH" >> ${HOME}/.zshenv
    echo eval "$(pyenv init --path)" >> ${HOME}/.zshenv
    echo eval "$(pyenv init -)" >> ${HOME}/.zshenv
    echo eval "$(pyenv-virtualenv init -)" >> ${HOME}/.zshenv
```